package.json

```diff
@@ -47,7 +47,9 @@
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.3.0",
-    "react-dom": "^16.3.2",
+    "fill-range": "^2.2.4",
+    "lodash": "^4.17.11",
+    "react-dom": "^16.3.3",
```

Comment:
Why not the latest React 16.8?

Reply:
since this is a public package, i didnt want to limit this for other consumers but you do bring up a good point that this change is for 16.4 and up so it should be ^16.4.0